content/en/internal/helpdesk/service-info/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Service information"
+linkTitle: "Service info"
+weight: 100
+type: "docs"
+draft: true
+description: >
+    Technical details of EGI helpdesk
+---
+
+## Identity card
+
+<!-- markdownlint-disable line-length no-bare-urls -->
+
+| Property                     | Value                                                                                               |
+| ---------------------------- | -------------------------------------------------------------------------------|
+| Name                         |                                                                                |
+| Description                  |                                                                                |
+| URL                          |                                                                                |
+| Support Email                |                                                                                |
+| [Helpdesk](..) Support Unit | EGI Services and Service components -> ?                                        |
+| Configuration Database entry |                                                                                |
+| Supplier                     |                                                                                |
+| Roadmap                      |                                                                                |
+| Release notes                |                                                                                |
+| Source code                  |                                                                                |
+| Issue tracker for developers |                                                                                |
+| License                      |                                                                                |
+| Privacy Policy               |                                                                                |
```

Comment:
Please align the headers/rows on the largest entry.

Reply:
Fixed